src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/ApplicationFormSectionsController.java

```diff
@@ -104,7 +104,7 @@ public ResponseEntity postNewSection(final HttpServletRequest request, @PathVari
         }
     }
 
-    @DeleteMapping("/{sectionId}")
+    @DeleteMapping("/{sectionId}/{version}")
```

Comment:
I think `/{sectionId}/version/{version}` or passing the version as a query param (`/sectionId?version=1`)might be slightly more RESTful. 

... sorry, I know this means a change to the front end as well

Reply:
query param is probably better - you're right. I'll make that change, thanks!